modules/rosa-about.adoc

```diff
@@ -0,0 +1,11 @@
+
+:system-module-type: REFERENCE
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/rosa-get-started-cli.adoc
+
+[id="rosa-about_{context}"]
+= About the `rosa` CLI
+
+[role="_abstract"]
+`rosa` is the the {product-title} (ROSA) command line utility. Use `rosa` to create, update, manage, and delete {product-title} clusters and resources.
```

Comment:
"the the"
Avoid starting a sentence with a lowercase word. Perhaps instead:
"With `rosa`, the command-line interface for {product-title} (ROSA), you can create, update...."

Reply:
Done.